File: wear/src/main/java/net/osdn/gokigen/joggingtimer/stopwatch/IClickCallback.java
package net.osdn.gokigen.joggingtimer.stopwatch;

import java.util.ArrayList;

public interface IClickCallback
{
    void clickedCounter();
    void clickedBtn1();
    void clickedBtn2();
    void clickedBtn3();

    boolean pushedBtn1();
    boolean pushedBtn2();
    boolean pushedBtn3();

    void dataIsReloaded(ArrayList<Long> list);

}

File: wear/src/main/java/net/osdn/gokigen/joggingtimer/stopwatch/IClickCallback.kt
package net.osdn.gokigen.joggingtimer.stopwatch

interface IClickCallback
{
    fun clickedCounter()
    fun clickedBtn1()
    fun clickedBtn2()
    fun clickedBtn3()
    fun clickedArea()
    fun pushedBtn1(): Boolean
    fun pushedBtn2(): Boolean
    fun pushedBtn3(): Boolean
    fun pushedArea(): Boolean
}
